src/index.js

```diff
@@ -1158,10 +1158,10 @@ module.exports = class WyzeAPI {
     await this.setProperty(deviceMac, deviceModel, "P3", value);
   }
   async lightTurnOn(deviceMac, deviceModel) {
-    await this.setProperty(deviceMac, deviceModel, "P3", "0");
+    await this.lightPower(deviceMac, deviceModel, "P3", "1");
```

Comment:
Why does this change to this.lightPower instead of setProperty

Reply:
this.lightPower is used in Homebridge Wyze Plugin. Its takes 3 arguments

Reply:
I was considering changing this to use lightPower but clearly didn't change things back ... just a typo / mistake on my part, thanks for cleaning it up!